packages/api/src/beacon/routes/events.ts

```diff
@@ -91,10 +91,10 @@ export type EventData = {
     block: RootHex;
     executionOptimistic: boolean;
   };
-  [EventType.attestation]: phase0.Attestation;
+  [EventType.attestation]: {version: ForkName; data: allForks.Attestation};
```

Comment:
good reminder still is still open on the spec, I don't think we can hack the existing format like this, might need `attestation_v2` or similar. I wish the initial design made them fork aware right away

Reply:
right, but this is just events so i guess its ok for now